digital-credentials/enabled-on-self-origin-by-permissions-policy.https.sub.html.headers

```diff
@@ -0,0 +1 @@
+Permissions-Policy: digital-credentials-get=(self)
```

Comment:
Optional: Add test for
Permissions-Policy: digital-credentials-get=*

Reply:
I might leave this one for now and do it as a followup. I need to implement support for `Permissions-Policy:` generally in WebKit still, so will need to do test it as part of that.